console/src/main/scripts/plugins/metrics/ts/app-details/appServerJvmDetails.ts

```diff
@@ -143,21 +143,57 @@ module HawkularMetrics {
     }
 
     private formatCounterChartOutput(response, buckets = 60):IChartDataPoint[] {
```

Comment:
This is something supposed to be (very) temporary, because once metrics have support for buckets in counters most of it will go away, the remaining will go with the ability of charts to handle metrics unmodified [bucket] data, as per https://issues.jboss.org/browse/HAWKULAR-643.

Still, this can go in the MetricsService, don't think there's anything specific to it that'd block it. Moving.